adapters/readpeak/readpeak_test.go

```diff
@@ -0,0 +1,21 @@
+package readpeak
+
+import (
+  "testing"
+
+  "github.com/prebid/prebid-server/v2/adapters/adapterstest"
+  "github.com/prebid/prebid-server/v2/config"
+  "github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+  bidder, buildErr := Builder(openrtb_ext.BidderReadpeak, config.Adapter{
+    Endpoint: "http://readpeaktest.com/bidder"},
```

Comment:
should use same endpoint mentioned in bidder-info yaml

Reply:
Updated the endpoint, although docs here https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html say you can use whatever URL.